homeassistant/components/formula_one/const.py

```diff
@@ -0,0 +1,11 @@
+"""Constants for the Formula 1 integration."""
+
+from datetime import timedelta
+
+DOMAIN = "formula_one"
+
+DEFAULT_UPDATE_INTERVAL = timedelta(hours=1)
+
+F1_STATE_MULTIPLE = "(multiple)"
+
+F1_DISCOVERY_NEW = "f1_discovery_new"
```

Comment:
This should be prefixed with the domain
```suggestion
F1_DISCOVERY_NEW = "formula_one_discovery_new"
```